lib/ansible/modules/windows/win_iis_webapppool.py

```diff
@@ -58,40 +58,118 @@
 '''
 
 EXAMPLES = r'''
-- name: Return information about an existing application pool
-  win_iis_webapppool:
-    name: DefaultAppPool
+# This returns information about an existing application pool
```

Comment:
I think you and @Fale are fighting here- IIRC he just went and removed a bunch of the cmdline-style examples from modules in favor of play/task-style docs. I'd tend to agree with showing playbook examples of these things over direct cmdline samples- unless there's a caveat to k=v conversion of a particular module arg, it sounds like most folks prefer looking at the YAML style. That's the direction we've been going with a lot of stuff lately...